<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="20" passed="11" failed="2" skipped="7">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-05-23T15:54:41 GMT" name="book-flight-module" finished-at="2023-05-23T16:04:22 GMT" duration-ms="580764">
    <groups>
    </groups>
    <test started-at="2023-05-23T15:54:41 GMT" name="book-flight-1" finished-at="2023-05-23T15:57:44 GMT" duration-ms="182744">
      <class name="com.newtours.tests.BookFlightTest">
        <test-method is-config="true" signature="setupValues(java.lang.String,java.lang.String)[pri:0, instance:com.newtours.tests.BookFlightTest@27c6e487]" started-at="2023-05-23T15:54:41 GMT" name="setupValues" finished-at="2023-05-23T15:57:43 GMT" duration-ms="182301" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[1]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[$584 USD]]>
              </value>
            </param>
          </params>
          <exception class="org.openqa.selenium.SessionNotCreatedException">
            <message>
              <![CDATA[Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: '8d32585f8abd', ip: '172.21.0.6'
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.104-linuxkit', java.version: '18-ea'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}], desiredCapabilities=Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}}]
Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: '8d32585f8abd', ip: '172.21.0.6'
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.104-linuxkit', java.version: '18-ea'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}], desiredCapabilities=Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}}]
Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:561)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:229)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:157)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:139)
at tests.baseClass.getDriver(baseClass.java:43)
at com.newtours.tests.BookFlightTest.setupValues(BookFlightTest.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:644)
at org.testng.TestRunner.beforeRun(TestRunner.java:633)
at org.testng.TestRunner.run(TestRunner.java:595)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
Caused by: java.lang.RuntimeException: NettyHttpHandler request execution error
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:76)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.execute(NettyHttpHandler.java:49)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyClient.execute(NettyClient.java:99)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:55)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:124)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:106)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:67)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:165)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
... 28 more
Caused by: java.util.concurrent.ExecutionException: java.util.concurrent.TimeoutException: Request timeout to hub/172.21.0.2:4444 after 180000 ms
at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096)
at org.asynchttpclient.netty.NettyResponseFuture.get(NettyResponseFuture.java:206)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:59)
... 41 more
Caused by: java.util.concurrent.TimeoutException: Request timeout to hub/172.21.0.2:4444 after 180000 ms
at org.asynchttpclient.netty.timeout.TimeoutTimerTask.expire(TimeoutTimerTask.java:43)
at org.asynchttpclient.netty.timeout.RequestTimeoutTimerTask.run(RequestTimeoutTimerTask.java:50)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.SessionNotCreatedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupValues -->
        <test-method signature="gotoRegistrationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@27c6e487]" started-at="2023-05-23T15:57:44 GMT" name="gotoRegistrationPage" finished-at="2023-05-23T15:57:44 GMT" duration-ms="0" status="SKIP">
          <exception class="org.openqa.selenium.SessionNotCreatedException">
            <message>
              <![CDATA[Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: '8d32585f8abd', ip: '172.21.0.6'
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.104-linuxkit', java.version: '18-ea'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}], desiredCapabilities=Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}}]
Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure. 
Host info: host: '8d32585f8abd', ip: '172.21.0.6'
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.104-linuxkit', java.version: '18-ea'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [null, newSession {capabilities=[Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}], desiredCapabilities=Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}}]
Capabilities {browserName: chrome, goog:chromeOptions: {args: [--remote-allow-origins=*], extensions: []}}
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:561)
at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:229)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:157)
at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:139)
at tests.baseClass.getDriver(baseClass.java:43)
at com.newtours.tests.BookFlightTest.setupValues(BookFlightTest.java:36)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:361)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:296)
at org.testng.TestRunner.invokeTestConfigurations(TestRunner.java:644)
at org.testng.TestRunner.beforeRun(TestRunner.java:633)
at org.testng.TestRunner.run(TestRunner.java:595)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
Caused by: java.lang.RuntimeException: NettyHttpHandler request execution error
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:76)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.execute(NettyHttpHandler.java:49)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyClient.execute(NettyClient.java:99)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:55)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:124)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:106)
at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:67)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:165)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
... 28 more
Caused by: java.util.concurrent.ExecutionException: java.util.concurrent.TimeoutException: Request timeout to hub/172.21.0.2:4444 after 180000 ms
at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096)
at org.asynchttpclient.netty.NettyResponseFuture.get(NettyResponseFuture.java:206)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:59)
... 41 more
Caused by: java.util.concurrent.TimeoutException: Request timeout to hub/172.21.0.2:4444 after 180000 ms
at org.asynchttpclient.netty.timeout.TimeoutTimerTask.expire(TimeoutTimerTask.java:43)
at org.asynchttpclient.netty.timeout.RequestTimeoutTimerTask.run(RequestTimeoutTimerTask.java:50)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.SessionNotCreatedException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gotoRegistrationPage -->
        <test-method signature="gotoRegistrationConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@27c6e487]" started-at="2023-05-23T15:57:44 GMT" name="gotoRegistrationConfirmationPage" finished-at="2023-05-23T15:57:44 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.gotoRegistrationPage" duration-ms="1" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.gotoRegistrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@27c6e487 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.gotoRegistrationPage() on instance com.newtours.tests.BookFlightTest@27c6e487]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.gotoRegistrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@27c6e487 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.gotoRegistrationPage() on instance com.newtours.tests.BookFlightTest@27c6e487]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gotoRegistrationConfirmationPage -->
        <test-method signature="flightDetailsPage()[pri:0, instance:com.newtours.tests.BookFlightTest@27c6e487]" started-at="2023-05-23T15:57:44 GMT" name="flightDetailsPage" finished-at="2023-05-23T15:57:44 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.gotoRegistrationConfirmationPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@27c6e487 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.gotoRegistrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@27c6e487]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@27c6e487 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.gotoRegistrationConfirmationPage() on instance com.newtours.tests.BookFlightTest@27c6e487]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightDetailsPage -->
        <test-method signature="findFlightPage()[pri:0, instance:com.newtours.tests.BookFlightTest@27c6e487]" started-at="2023-05-23T15:57:44 GMT" name="findFlightPage" finished-at="2023-05-23T15:57:44 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.flightDetailsPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@27c6e487 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@27c6e487]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@27c6e487 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@27c6e487]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findFlightPage -->
        <test-method signature="FlightConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@27c6e487]" started-at="2023-05-23T15:57:44 GMT" name="FlightConfirmationPage" finished-at="2023-05-23T15:57:44 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.findFlightPage" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.FlightConfirmationPage() on instance com.newtours.tests.BookFlightTest@27c6e487 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@27c6e487]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.FlightConfirmationPage() on instance com.newtours.tests.BookFlightTest@27c6e487 depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@27c6e487]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FlightConfirmationPage -->
      </class> <!-- com.newtours.tests.BookFlightTest -->
    </test> <!-- book-flight-1 -->
    <test started-at="2023-05-23T15:57:44 GMT" name="book-flight-2" finished-at="2023-05-23T16:04:01 GMT" duration-ms="376990">
      <class name="com.newtours.tests.BookFlightTest">
        <test-method is-config="true" signature="setupValues(java.lang.String,java.lang.String)[pri:0, instance:com.newtours.tests.BookFlightTest@7f77e91b]" started-at="2023-05-23T15:57:44 GMT" name="setupValues" finished-at="2023-05-23T16:00:07 GMT" duration-ms="142323" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[2]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[$1169 USD]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupValues -->
        <test-method signature="gotoRegistrationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@7f77e91b]" started-at="2023-05-23T16:00:07 GMT" name="gotoRegistrationPage" finished-at="2023-05-23T16:00:13 GMT" duration-ms="6725" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gotoRegistrationPage -->
        <test-method signature="gotoRegistrationConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@7f77e91b]" started-at="2023-05-23T16:00:13 GMT" name="gotoRegistrationConfirmationPage" finished-at="2023-05-23T16:00:14 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.gotoRegistrationPage" duration-ms="448" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gotoRegistrationConfirmationPage -->
        <test-method signature="flightDetailsPage()[pri:0, instance:com.newtours.tests.BookFlightTest@7f77e91b]" started-at="2023-05-23T16:00:14 GMT" name="flightDetailsPage" finished-at="2023-05-23T16:04:01 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.gotoRegistrationConfirmationPage" duration-ms="227337" status="FAIL">
          <exception class="org.openqa.selenium.remote.UnreachableBrowserException">
            <message>
              <![CDATA[Error communicating with the remote browser. It may have died.
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.104-linuxkit', java.version: '18-ea'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [b7fec6116039ea6ad538dd20b284380a, findElement {using=id, value=passCount}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 96.0.4664.93, chrome: {chromedriverVersion: 96.0.4664.45 (76e4c1bb2ab46..., userDataDir: /tmp/.com.google.Chrome.BtfGKy}, goog:chromeOptions: {debuggerAddress: localhost:41885}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:cdp: ws://172.21.0.3:4444/sessio..., se:cdpVersion: 96.0.4664.93, se:vnc: ws://172.21.0.3:4444/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.21.0.3:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: b7fec6116039ea6ad538dd20b284380a]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Error communicating with the remote browser. It may have died.
Build info: version: '4.8.3', revision: 'e5e76298c3'
System info: os.name: 'Linux', os.arch: 'amd64', os.version: '5.10.104-linuxkit', java.version: '18-ea'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Command: [b7fec6116039ea6ad538dd20b284380a, findElement {using=id, value=passCount}]
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 96.0.4664.93, chrome: {chromedriverVersion: 96.0.4664.45 (76e4c1bb2ab46..., userDataDir: /tmp/.com.google.Chrome.BtfGKy}, goog:chromeOptions: {debuggerAddress: localhost:41885}, networkConnectionEnabled: false, pageLoadStrategy: normal, platformName: LINUX, proxy: Proxy(), se:cdp: ws://172.21.0.3:4444/sessio..., se:cdpVersion: 96.0.4664.93, se:vnc: ws://172.21.0.3:4444/sessio..., se:vncEnabled: true, se:vncLocalAddress: ws://172.21.0.3:7900, setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: b7fec6116039ea6ad538dd20b284380a
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:567)
at org.openqa.selenium.remote.ElementLocation$ElementFinder$2.findElement(ElementLocation.java:162)
at org.openqa.selenium.remote.ElementLocation.findElement(ElementLocation.java:60)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:352)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:344)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:70)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:39)
at jdk.proxy2/jdk.proxy2.$Proxy13.isEnabled(Unknown Source)
at org.openqa.selenium.support.ui.Select.assertSelectIsEnabled(Select.java:331)
at org.openqa.selenium.support.ui.Select.selectByValue(Select.java:193)
at com.newtours.pages.FlightDetailsPage.selectPassengers(FlightDetailsPage.java:38)
at com.newtours.tests.BookFlightTest.flightDetailsPage(BookFlightTest.java:66)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
Caused by: java.lang.RuntimeException: NettyHttpHandler request execution error
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:76)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.execute(NettyHttpHandler.java:49)
at org.openqa.selenium.remote.http.AddSeleniumUserAgent.lambda$apply$0(AddSeleniumUserAgent.java:42)
at org.openqa.selenium.remote.http.Filter.lambda$andFinally$1(Filter.java:56)
at org.openqa.selenium.remote.http.netty.NettyClient.execute(NettyClient.java:99)
at org.openqa.selenium.remote.tracing.TracedHttpClient.execute(TracedHttpClient.java:55)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:190)
at org.openqa.selenium.remote.TracedCommandExecutor.execute(TracedCommandExecutor.java:51)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:543)
... 38 more
Caused by: java.util.concurrent.ExecutionException: java.util.concurrent.TimeoutException: Request timeout to hub/172.21.0.2:4444 after 180000 ms
at java.base/java.util.concurrent.CompletableFuture.reportGet(CompletableFuture.java:396)
at java.base/java.util.concurrent.CompletableFuture.get(CompletableFuture.java:2096)
at org.asynchttpclient.netty.NettyResponseFuture.get(NettyResponseFuture.java:206)
at org.openqa.selenium.remote.http.netty.NettyHttpHandler.makeCall(NettyHttpHandler.java:59)
... 48 more
Caused by: java.util.concurrent.TimeoutException: Request timeout to hub/172.21.0.2:4444 after 180000 ms
at org.asynchttpclient.netty.timeout.TimeoutTimerTask.expire(TimeoutTimerTask.java:43)
at org.asynchttpclient.netty.timeout.RequestTimeoutTimerTask.run(RequestTimeoutTimerTask.java:50)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.run(HashedWheelTimer.java:715)
at io.netty.util.concurrent.ImmediateExecutor.execute(ImmediateExecutor.java:34)
at io.netty.util.HashedWheelTimer$HashedWheelTimeout.expire(HashedWheelTimer.java:703)
at io.netty.util.HashedWheelTimer$HashedWheelBucket.expireTimeouts(HashedWheelTimer.java:790)
at io.netty.util.HashedWheelTimer$Worker.run(HashedWheelTimer.java:503)
at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
at java.base/java.lang.Thread.run(Thread.java:833)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.remote.UnreachableBrowserException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightDetailsPage -->
        <test-method signature="findFlightPage()[pri:0, instance:com.newtours.tests.BookFlightTest@7f77e91b]" started-at="2023-05-23T16:04:01 GMT" name="findFlightPage" finished-at="2023-05-23T16:04:01 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.flightDetailsPage" duration-ms="1" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@7f77e91b depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@7f77e91b]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@7f77e91b depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.flightDetailsPage() on instance com.newtours.tests.BookFlightTest@7f77e91b]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findFlightPage -->
        <test-method signature="FlightConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@7f77e91b]" started-at="2023-05-23T16:04:01 GMT" name="FlightConfirmationPage" finished-at="2023-05-23T16:04:01 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.findFlightPage" duration-ms="1" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method com.newtours.tests.BookFlightTest.FlightConfirmationPage() on instance com.newtours.tests.BookFlightTest@7f77e91b depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@7f77e91b]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method com.newtours.tests.BookFlightTest.FlightConfirmationPage() on instance com.newtours.tests.BookFlightTest@7f77e91b depends on not successfully finished methods 
[com.newtours.tests.BookFlightTest.findFlightPage() on instance com.newtours.tests.BookFlightTest@7f77e91b]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:150)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FlightConfirmationPage -->
      </class> <!-- com.newtours.tests.BookFlightTest -->
    </test> <!-- book-flight-2 -->
    <test started-at="2023-05-23T16:04:02 GMT" name="book-flight-3" finished-at="2023-05-23T16:04:14 GMT" duration-ms="12636">
      <class name="com.newtours.tests.BookFlightTest">
        <test-method is-config="true" signature="setupValues(java.lang.String,java.lang.String)[pri:0, instance:com.newtours.tests.BookFlightTest@2357d90a]" started-at="2023-05-23T16:04:02 GMT" name="setupValues" finished-at="2023-05-23T16:04:02 GMT" duration-ms="9" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[3]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[$1753 USD]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupValues -->
        <test-method signature="gotoRegistrationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2357d90a]" started-at="2023-05-23T16:04:02 GMT" name="gotoRegistrationPage" finished-at="2023-05-23T16:04:10 GMT" duration-ms="8425" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gotoRegistrationPage -->
        <test-method signature="gotoRegistrationConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2357d90a]" started-at="2023-05-23T16:04:10 GMT" name="gotoRegistrationConfirmationPage" finished-at="2023-05-23T16:04:11 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.gotoRegistrationPage" duration-ms="389" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gotoRegistrationConfirmationPage -->
        <test-method signature="flightDetailsPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2357d90a]" started-at="2023-05-23T16:04:11 GMT" name="flightDetailsPage" finished-at="2023-05-23T16:04:13 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.gotoRegistrationConfirmationPage" duration-ms="2613" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightDetailsPage -->
        <test-method signature="findFlightPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2357d90a]" started-at="2023-05-23T16:04:13 GMT" name="findFlightPage" finished-at="2023-05-23T16:04:14 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.flightDetailsPage" duration-ms="768" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findFlightPage -->
        <test-method signature="FlightConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@2357d90a]" started-at="2023-05-23T16:04:14 GMT" name="FlightConfirmationPage" finished-at="2023-05-23T16:04:14 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.findFlightPage" duration-ms="393" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FlightConfirmationPage -->
      </class> <!-- com.newtours.tests.BookFlightTest -->
    </test> <!-- book-flight-3 -->
    <test started-at="2023-05-23T16:04:15 GMT" name="book-flight-4" finished-at="2023-05-23T16:04:22 GMT" duration-ms="7168">
      <class name="com.newtours.tests.BookFlightTest">
        <test-method is-config="true" signature="setupValues(java.lang.String,java.lang.String)[pri:0, instance:com.newtours.tests.BookFlightTest@8b96fde]" started-at="2023-05-23T16:04:15 GMT" name="setupValues" finished-at="2023-05-23T16:04:15 GMT" duration-ms="2" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[4]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[$1169 USD]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setupValues -->
        <test-method signature="gotoRegistrationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@8b96fde]" started-at="2023-05-23T16:04:15 GMT" name="gotoRegistrationPage" finished-at="2023-05-23T16:04:19 GMT" duration-ms="4201" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gotoRegistrationPage -->
        <test-method signature="gotoRegistrationConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@8b96fde]" started-at="2023-05-23T16:04:19 GMT" name="gotoRegistrationConfirmationPage" finished-at="2023-05-23T16:04:19 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.gotoRegistrationPage" duration-ms="218" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- gotoRegistrationConfirmationPage -->
        <test-method signature="flightDetailsPage()[pri:0, instance:com.newtours.tests.BookFlightTest@8b96fde]" started-at="2023-05-23T16:04:19 GMT" name="flightDetailsPage" finished-at="2023-05-23T16:04:21 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.gotoRegistrationConfirmationPage" duration-ms="1828" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- flightDetailsPage -->
        <test-method signature="findFlightPage()[pri:0, instance:com.newtours.tests.BookFlightTest@8b96fde]" started-at="2023-05-23T16:04:21 GMT" name="findFlightPage" finished-at="2023-05-23T16:04:21 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.flightDetailsPage" duration-ms="636" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- findFlightPage -->
        <test-method signature="FlightConfirmationPage()[pri:0, instance:com.newtours.tests.BookFlightTest@8b96fde]" started-at="2023-05-23T16:04:21 GMT" name="FlightConfirmationPage" finished-at="2023-05-23T16:04:22 GMT" depends-on-methods="com.newtours.tests.BookFlightTest.findFlightPage" duration-ms="269" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [$1169 USD] but found [$2338 USD]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [$1169 USD] but found [$2338 USD]
at org.testng.Assert.fail(Assert.java:110)
at org.testng.Assert.failNotEquals(Assert.java:1413)
at org.testng.Assert.assertEqualsImpl(Assert.java:149)
at org.testng.Assert.assertEquals(Assert.java:131)
at org.testng.Assert.assertEquals(Assert.java:655)
at org.testng.Assert.assertEquals(Assert.java:665)
at com.newtours.tests.BookFlightTest.FlightConfirmationPage(BookFlightTest.java:80)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:677)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:221)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:50)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:969)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:194)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:148)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:829)
at org.testng.TestRunner.run(TestRunner.java:602)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:437)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:431)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:391)
at org.testng.SuiteRunner.run(SuiteRunner.java:330)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1256)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1176)
at org.testng.TestNG.runSuites(TestNG.java:1099)
at org.testng.TestNG.run(TestNG.java:1067)
at org.testng.TestNG.privateMain(TestNG.java:1414)
at org.testng.TestNG.main(TestNG.java:1378)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- FlightConfirmationPage -->
      </class> <!-- com.newtours.tests.BookFlightTest -->
    </test> <!-- book-flight-4 -->
  </suite> <!-- book-flight-module -->
</testng-results>
